tests/test_user_remote_storage.rs

```diff
@@ -0,0 +1,93 @@
+use colink::{
+    extensions::instant_server::{InstantRegistry, InstantServer},
+    CoLink, SubscriptionMessage,
+};
+
+async fn user_remote_storage_fn(
+    cla: &CoLink,
+    clb: &CoLink,
+    msg: &str,
+) -> Result<(), Box<dyn std::error::Error + Send + Sync + 'static>> {
+    let user_id_a = cla.get_user_id().unwrap();
+    let user_id_b = clb.get_user_id().unwrap();
+
+    // create
+    cla.remote_storage_create(
+        &[user_id_b.clone()],
+        "remote_storage_demo",
+        msg.as_bytes(),
+        false,
+    )
+    .await?;
+
+    let data = clb
+        .read_or_wait(&format!(
+            "_remote_storage:private:{}:remote_storage_demo",
+            user_id_a
+        ))
+        .await?;
+    // println!("{}", String::from_utf8_lossy(&data));
```

Comment:
remove comments? or uncommenting them?

Reply:
Actually, I think we should make assertions in a test, rather than manually checking the printing results?